src/routes/startup_profile.js

```diff
@@ -0,0 +1,11 @@
+module.exports = (req, res)=> {
+  res.render('startup_profile', {
+    founderName: 'Mario',
+    startupName: 'FAC',
+    email: 'mario_the_freak@gmail.com',
+    phoneNumber: 05236521562,
+    websiteLink: 'foundersandcoders.com',
+    linkedinLink: 'www.linkdin.com',
```

Comment:
maybe add a facebook link and make social media into another object so it would be less messy